docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+.. _supported_tasks:
```

Comment:
labels have to be unique within a project...
```suggestion
.. _nlp_supported_tasks:
```

Reply:
by the way - why did you add these all throughout? (just wondering cause I didn't see where they were used)

Reply:
I just copied the way the rest of the docs are built...